source/administration/changelog.md

```diff
@@ -4,6 +4,97 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
 
 Also see [changelog in progress](https://bit.ly/2nK3cVf) for the next release.
 
+## Release v5.34 - [Feature Release](https://docs.mattermost.com/administration/release-definitions.html#feature-release)
+
+**Release Day: 2021-04-16**
+
+### Combatibility
+
+### Breaking Changes
+ 
+**IMPORTANT:** If you upgrade from a release earlier than v5.33, please read the other [Important Upgrade Notes](https://docs.mattermost.com/administration/important-upgrade-notes.html).
+
+### Highlights
+
+#### Added pre-packaged Incident Collaboration 1.5.0 (E20)
+ - Users can now configure playbooks to perform specific actions automatically when an incident starts, such as inviting members to the incident channel and assigning a commander.
+
+### Improvements
+
+#### User Interface (UI)
+ - Removed Beta tags from Swedish and Bulgarian languages.
+ - System Admins are now prompted when joining a private channel via a permalink.
```

Comment:
Prompted for what, exactly? To confirm that they wish to join the private channel?

Reply:
Fixed the text.